### PR TITLE
Create Calculator components and layout

### PR DESCRIPTION
**Pull request by GutemaG**
## Create Calculator components and layout
- Create ```src/components```
- Create class based component ``` Calculator ```;
- Create layout for math-magicians calculator layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (KDlamini) [inline src/App.js]: Kindly make sure that the `App.js` component is class-based as per project requirements. Although you'll eventually refactor your components to using Hooks, it is important to learn how to use both class and functional components. For now, you'll be learning how to use class-based components until further notice.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @GutemaG 🙋‍♀️,  

Great work so far! 💯 
You have done a great job initiating the Math magicians app with components. 🎊 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  The project should have a directory called `components`
✔️  The `Calculator.js` component is class-based
✔️  The `App.js`(main component) renders the `Calculator.js` component
✔️  No linter errors

## Required Changes ♻️

- Kindly remove the comment between line 3 to line 26 in your README to make it cleaner and more professional.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (GutemaG) [reply src/App.js]: Thank you @KDlamini

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @GutemaG 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GutemaG** [comment]
Thank you @HAMMAS-SALEEM !

**Pull request merged**